pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -628,6 +594,61 @@ func filterRemaining(kvs []streampb.StreamEvent_KV) []streampb.StreamEvent_KV {
 	return remaining[:j]
 }
 
+func (lrw *logicalReplicationWriterProcessor) setupBatchHandlers(ctx context.Context) error {
+	if lrw.FlowCtx == nil {
+		return nil
+	}
```

Comment:
If we hit this case, seems like we'd fail elsewhere that expects lhr.bh to be non-nil?

Reply:
bh is non-nil in the unit tests; it is just FlowCtx they don’t setup